packages/plugin-e2e/src/models/pages/AppPage.ts

```diff
@@ -0,0 +1,18 @@
+import * as nodePath from 'path';
+import { NavigateOptions, PluginPageArgs, PluginTestCtx } from '../../types';
+import { GrafanaPage } from '..';
+
+export class AppPage extends GrafanaPage {
+  constructor(readonly ctx: PluginTestCtx, readonly args: PluginPageArgs) {
+    super(ctx);
+  }
+
+  /**
+   * Will append the `/a/${pluginId}` before the provided path and then
+   * navigate to the page.
+   */
+  goto(path?: string, options?: NavigateOptions): Promise<void> {
```

Comment:
In order to avoid breaking changes, I've been trying to use objects rather than primitives for all public methods. What do you think about changing this to
```ts
goto(options?: NavigateOptions & { path?: string }): Promise<void> {
```